rest/config_test.go

```diff
@@ -887,3 +888,41 @@ func TestPutInvalidConfig(t *testing.T) {
 	response := rt.SendAdminRequest("PUT", "/db/_config", `{"db": {"server": "walrus"}}`)
 	assert.Equal(t, http.StatusBadRequest, response.Code)
 }
+
+// Validate basic mapping from config to database options
+func TestConfigToDatabaseOptions(t *testing.T) {
+
+	bucket := base.GetTestBucket(t)
+	defer bucket.Close()
+
+	bucketUser, bucketPassword, _ := bucket.BucketSpec.Auth.GetCredentials()
+	spec := bucket.BucketSpec
+
+	jsonConfig := []byte(`
+{
+	"databases": {
+		"db": {
+			"server": "` + spec.Server + `",
+			"username": "` + bucketUser + `",
+			"password": "` + bucketPassword + `",
+			"bucket": "` + spec.BucketName + `",
```

Comment:
It's kinda annoying, but we'll have to add these extra options for this to work with our integration test setup.

```suggestion
			"bucket": "` + spec.BucketName + `",
			"use_views": ` + strconv.FormatBool(base.TestsDisableGSI()) + `,
			"num_index_replicas": 0,
```

Reply:
Ok thanks